### DISCUSSION TITLE
its not working in deepin mangaro ...

### DISCUSSION THREAD

**Discussion by ghost** [linuxdeepin/developer-center#3040] [Q&A | 问答板块] [answered]
how to install deepin recovery ..
![DeepinScreenshot_20190915112445](https://user-images.githubusercontent.com/34711035/64918453-1bb33c00-d7be-11e9-9385-e3279d1fe221.png)
![DeepinScreenshot_20190915133426](https://user-images.githubusercontent.com/34711035/64918455-1d7cff80-d7be-11e9-8fde-ba034152dd9c.png)

**Comment by marcelosenabr** [comment, marked as answer]
> como instalar o deepin recovery ..
> ![DeepinScreenshot_20190915112445](https://user-images.githubusercontent.com/34711035/64918453-1bb33c00-d7be-11e9-9385-e3279d1fe221.png)
> ![DeepinScreenshot_20190915133426](https://user-images.githubusercontent.com/34711035/64918455-1d7cff80-d7be-11e9-8fde-ba034152dd9c.png)

Look here
https://youtu.be/sYFFeakdSfM

And 

https://github.com/linuxdeepin/deepin-clone/issues/16
